### ISSUE TITLE
Voiceover

### ISSUE THREAD

**Issue by MichaelaRob** [sander-aaron/ecommerce-pattern-library#7]
Add some aria details to make voiceover more accessible

**Comment by maddycpierce** [comment]
Product details page: unable to select the size or "Add to your order" button when using voiceover
Check out page: unable to select buttons
